contracts/truefi/TrueLender.sol

```diff
@@ -77,6 +77,9 @@ contract TrueLender is ITrueLender, Ownable {
     // maximum amount of loans lender can handle at once
     uint256 public maxLoans;
 
+    // implemented as an ERC20, will change after implementing stkPool
+    IERC20 public stakingPool;
```

Comment:
Is there a reason, we can't keep it as a simple address?

Reply:
You're right, thank You for that :)